class-vip-support-user.php

```diff
@@ -629,6 +629,38 @@ protected function get_trigger_resend_verification_url() {
 		return add_query_arg( array( self::GET_TRIGGER_RESEND_VERIFICATION => '1' ) );
 	}
 
+	/**
+	 * Get list of emails for current Mentees from Zendesk
+	 *
+	 * @return array Array of emails for current Mentees
+	 */
+	protected function get_mentees() {
+		$cached_emails = wpcom_vip_cache_get( 'vip_mentees' );
+
+		if ( $cached_emails ) {
```

Comment:
If we had cached an empty array (nobody in the group), this check would not pass (empty array evaluates to false in PHP), and we'd end up running the uncached query on every request.

Better to use a strict check - `if ( false !== $cached_emails )`. We should also make sure it's an array, as later code expects it to be (in case some other weird value made it in there).